src/sentry/static/sentry/app/components/stream/group.jsx

```diff
@@ -133,13 +137,19 @@ const StreamGroup = createReactClass({
           <GroupChart id={data.id} statsPeriod={this.props.statsPeriod} data={data} />
         </div>
         <div className="col-md-1 col-xs-2 event-count align-right">
+          {hasGuideAnchor ? <GuideAnchor target="events" type="text" /> : null}
           <Count value={data.count} />
         </div>
         <div className="col-md-1 col-xs-2 event-users align-right">
+          {hasGuideAnchor ? <GuideAnchor target="users" type="text" /> : null}
           <Count value={userCount} />
         </div>
       </li>
     );
   },
 });
+
+const StyledGuideAnchor = styled(GuideAnchor)`
+  display: inline;
```

Comment:
Hmm ideally we don't want people having to do one-off overrides to make guide anchors work. If this is needed, we should figure out how to export another type of anchor from the library that can be used as is, and document it.

Reply:
Agree, let's move all styling into the GuideAnchor component

Reply:
Ah yes so this is temporary. I'll add a comment next to it. This is a stop gap - chrissy will take this out when he submits his next iteration of design for assistant. It was easier to do this and ship the first iteration.

Reply:
tested stuff out.. if you make the GuideAnchor the first element inside the `<div className="checkbox">` it should work